module.go

```diff
@@ -119,14 +119,21 @@ func NewModuleFromBufferArgs(args NewModuleArgs) (*Module, error) {
 		return nil, err
 	}
 
-	if args.BTFObjPath == "" {
-		args.BTFObjPath = "/sys/kernel/btf/vmlinux"
```

Comment:
I suppose we should keep the hard-coded path when not set by the caller.

Reply:
It should have the same behavior as `NewModuleFromFileArgs` ( see https://github.com/aquasecurity/libbpfgo/blob/main/module.go#L71).

It is currently not possible to set no BTFObjPath.